agent/config/config.go

```diff
@@ -177,6 +177,7 @@ type Config struct {
 	EnableAgentTLSForChecks          *bool               `mapstructure:"enable_agent_tls_for_checks" json:"enable_agent_tls_for_checks,omitempty"`
 	EnableCentralServiceConfig       *bool               `mapstructure:"enable_central_service_config" json:"enable_central_service_config,omitempty"`
 	EnableDebug                      *bool               `mapstructure:"enable_debug" json:"enable_debug,omitempty"`
+	EnableDevResources               *bool               `mapstructure:"enable_dev_resources" json:"enable_dev_resources,omitempty"`
```

Comment:
I was getting ready to implement a generic feature flag set in the config. Something like:

```
features = [
   "catalog-v2-alpha",
]
```

Do you think this dev resource enablement would fit better into that framework?

Reply:
Sounds good to me!